saleor/core/utils/random_data.py

```diff
@@ -200,23 +198,22 @@ def create_categories(categories_data, placeholder_dir):
         pk = category["pk"]
         defaults = category["fields"]
         parent = defaults["parent"]
-        image_name = (
-            CATEGORY_IMAGES[pk] if pk in CATEGORY_IMAGES else CATEGORY_IMAGES[parent]
-        )
-        background_image = get_image(placeholder_dir, image_name)
-        defaults["background_image"] = background_image
+        image_name = CATEGORY_IMAGES[pk] if pk in CATEGORY_IMAGES else None
```

Comment:
CATEGORY_IMAGES is a dict. instead of doing:
`CATEGORY_IMAGES[pk] if pk in CATEGORY_IMAGES else None`
I think that you could simply call `CATEGORY_IMAGES.get(pk)`

Reply:
Fixed.